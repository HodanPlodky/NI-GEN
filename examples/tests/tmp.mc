int main() {
    int i = 1;
    int j = 10;
    int * pi = &i;
    int x = i + j;
    i = 56;
    *pi = 123;
    return i;
}
