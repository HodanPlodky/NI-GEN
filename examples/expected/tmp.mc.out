input file : examples/tests/tmp.mc
return 123
